pkg/instancegroups/instancegroups.go

```diff
@@ -148,76 +149,49 @@ func (r *RollingUpdateInstanceGroup) RollingUpdate(rollingUpdateData *RollingUpd
 		}
 	}
 
-	for _, u := range update {
-		instanceId := u.ID
+	settings := resolveSettings(cluster, r.CloudGroup.InstanceGroup, numInstances)
 
-		nodeName := ""
-		if u.Node != nil {
-			nodeName = u.Node.Name
-		}
+	concurrency := 0
```

Comment:
Nit: `runningTasks` or similar might be a clearer name than `concurrency`

Reply:
Went with `runningDrains`